v22.2/performance-recipes.md

```diff
@@ -167,6 +172,22 @@ ORDER BY total_reads ASC;
 
 Use the values in the `total_reads` and `last_read` columns to identify indexes that have low usage or are stale and could be dropped.
 
+### Too many MVCC versions
```

Comment:
Should this be MVCC ( multi-**version** concurrency control) values (not "versions")?

Other places in our docs seem to refer to this as "MVCC values".

Reply:
Fixed.